cmd/config.go

```diff
@@ -95,6 +95,16 @@ $ juicefs config redis://localhost --min-client-version 1.0.0 --max-client-versi
 				Name:  "force",
 				Usage: "skip sanity check and force update the configurations",
 			},
+			&cli.Int64Flag{
+				Name:  "volume-upload-limit",
```

Comment:
It's the default value for this volume, we should call it `default-upload-limit`, or just `upload-limit`